lib/orbf/rules_engine/data/pyramid.rb

```diff
@@ -27,15 +28,23 @@ def groupset(ext_id)
         org_unit_groupsets_by_id[ext_id]
       end
 
+      def groups(ext_ids)
+        ext_ids.map { |ext_id| org_unit_groups_by_id[ext_id] }
```

Comment:
most specs fails because, looks like fetch_values raise KeyError: key not found: ["G_ID_1"]
and the fixtures are inconsistent.

don't know what is best... failing fast is better. but the 3 calls to dhis2 in order to fetch orgunit, groups groupsets might have inconistencies between them and so might fails fast also.